media-libs/opencollada/opencollada-1.6.68.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils eutils flag-o-matic
+
+DESCRIPTION="Stream based read/write library for COLLADA files"
+HOMEPAGE="http://www.opencollada.org/"
+SRC_URI="https://github.com/KhronosGroup/OpenCOLLADA/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+IUSE="static-libs"
+
+RDEPEND="dev-libs/libpcre
+	dev-libs/libxml2
+	dev-libs/zziplib
+	media-libs/lib3ds
+	sys-libs/zlib
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	app-admin/chrpath
+"
+
+S="${WORKDIR}/OpenCOLLADA-${PV}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-null-conversion.patch"
+	"${FILESDIR}/${P}-cmake-fixes.patch"
+	"${FILESDIR}/${PN}-1.6.63-pcre-fix.patch"
+)
+
+src_prepare() {
+	edos2unix CMakeLists.txt
+
+	cmake-utils_src_prepare
+
+	# Remove bundled depends that have portage equivalents
+	rm -rv Externals/{expat,lib3ds,LibXML,pcre,zziplib} || die
+
+	# Remove unused build systems
+	rm -v Makefile scripts/{unixbuild.sh,vcproj2cmake.rb} || die
+	find "${S}" -name SConscript -delete || die
+}
+
+src_configure() {
+	# bug 619670
+	append-cxxflags -std=c++14
+
+	local mycmakeargs=(
+		-DUSE_SHARED=ON
+		-DUSE_STATIC=$(usex static-libs)
+		-DUSE_LIBXML=ON
+	)
+
+	cmake-utils_src_configure
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	echo "LDPATH=/usr/$(get_libdir)/opencollada" > "${T}"/99${PN} || die "echo failed"
+	doenvd "${T}"/99${PN}
```

Comment:
Can be done with a simple `newenvd`. See https://gitweb.gentoo.org/repo/gentoo.git/tree/mail-client/thunderbird-bin/thunderbird-bin-68.8.0.ebuild#n151 for example.